html/files/knowledge_panel_test.json

```diff
@@ -0,0 +1,92 @@
+{
+    "doyouknow_brands_nutella_423": {
+        "type": "doyouknow",
+        "level": "trivia",
+        "topics": [
+            "ingredients"
+        ],
+        "title": "Do you know why Nutella contains hazelnuts?",
+        "subtitle": "It all started after the second world war...",
+        "elements": [
+            {
+                "element_type": "text",
+                "element": {
+                    "html": "Cocoa beans were expensive and hard to come by after the second world war, so in Piedmont (Italy) where Pietro Ferrero created Nutella, they were replaced with hazelnuts to make <em>gianduja</em>, a mix of hazelnut paste and chocolate."
+                }
+            }
+        ]
+    },
+    "ecoscore": {
+        "type": "ecoscore",
+        "level": "info",
+        "topics": [
+            "environment"
+        ],
+        "title": "Eco-Score D",
+        "subtitle": "High environmental impact",
+        "grade": "d",
+        "icon_url": "https://static.openfoodfacts.org/images/attributes/ecoscore-d.svg",
+        "elements": [
+            {
+                "element_type": "text",
+                "element": {
```

Comment:
can you add an image element as well ?

Reply:
Sure, I added an image to the first "doyouknow" panel.

Reply:
Thank you